app/assets/stylesheets/legacy_assets/modules/_notifications.css.scss

```diff
@@ -32,10 +32,10 @@
 
   .notice {
     a {
-      color: $primary-color;
+      color: white;
```

Comment:
Is there a sass variable for a white or whitish color in this gem? I think it's preferred that we use that if it exists. If not, no worries.